### ISSUE TITLE
Catalina island data issues

### ISSUE THREAD

**Issue by maning** [osmlab/labuildings#110]
While [validating Catalina Island](http://labuildingsimport.com/project/9#task/411), I noticed some data issues.  Not directly actionable in this import but logging here for future actions on cleanup. 

![screen shot 2016-09-17 at 10 50 44](https://cloud.githubusercontent.com/assets/353700/18606701/f35ab1bc-7cd4-11e6-84f8-15a2478f4c9f.png)
*Different names for roads in OSM and LACountry TMS*

![screen shot 2016-09-17 at 10 46 34](https://cloud.githubusercontent.com/assets/353700/18606700/f35987a6-7cd4-11e6-8c84-a3b21d6589d4.png)
*Imported addresses doesn't align very well with building, see west side of the image*

![screen shot 2016-09-17 at 10 45 14](https://cloud.githubusercontent.com/assets/353700/18606702/f35b348e-7cd4-11e6-95b7-1e457d36a6c1.png)
*Imported buildings were not split to individual house*

**Comment by jschleuss** [comment]
@maning Yep! There are some peculiar things with the island. The [trip in two weeks](http://www.meetup.com/MaptimeLA/events/233538372/) will attempt to fix some of these quirks. The city, county and fire stations admit that addresses are a problem. We're hoping to fix the 1,100 of those. And those buildings...I debate...should they be split? Those dividing walls are pretty wide. Maybe they should?

The local fire station has agreed to host us and we'll split into teams to do these things. Let me know if you think there are other pressing issues. Trip meant to grow local OSM community and fix issues in a relatively small area:

- **Address cleanup/businesses:** About 1,100 addresses need to be verified. Several more need to be added. All those imported in Avalon got a maptimela:reviewed=no tag for verification. Will need to make sure teams work on separate streets. Move address points to main entrance of structure (or entrance associated with address). Teams of 2. Instruction page
- **Botanic garden:** Add plants and trails using mobile apps or paper maps. 2 or 3 people total (including one person with a 360 camera for Mapillary).
- **Mapillary:** Will walk streets of Avalon collecting 360 imagery. One person in Botanic Gardens and one team of two will walk the circle trail above the gardens and Hermit Gulch.
- **Small feature check/add:** fire hydrants, benches, waste bins, ATMs, pay phones, bike parking, drinking fountains, decorative water fountains, tram stops.
- **Trees:** for anyone botanically inclined. Species name. Might be handy for the city or fire dept. Could be aided by command center.
- **Historical monuments:** List from the island museum. Museum may also let us review their maps to see what we should add to OSM.
- **Tourism tags:** artwork, attractions, camp sites, gallery, hotels, motels, information centers, picnic sites and viewpoints. More on OSM wiki
- **Verify/coding/central command:** group at fire station to review edits and provide verification. 
- **Sidewalks:** Following UW’s recently proposed schema
- **Campsite:** Add features of the Hermit Gulch Campground including the ref=campsite number following these proposed tags
